pkg/util/node/node.go

```diff
@@ -95,13 +95,25 @@ func PatchNodeCIDRs(c clientset.Interface, node types.NodeName, cidrs []string)
 
 // PatchNodeMultiNetwork patches the Node's annotations and capacity for MN.
 func PatchNodeMultiNetwork(c clientset.Interface, node *v1.Node) error {
+	annotation := make(map[string]string)
+	if val, ok := node.Annotations[networkv1.NorthInterfacesAnnotationKey]; ok {
+		annotation[networkv1.NorthInterfacesAnnotationKey] = val
+	}
+	if val, ok := node.Annotations[networkv1.MultiNetworkAnnotationKey]; ok {
+		annotation[networkv1.MultiNetworkAnnotationKey] = val
+	}
+
+	if len(annotation) > 0 {
+		raw, err := json.Marshal(annotation)
+		if err != nil {
+			return fmt.Errorf("failed to build patch bytes for multi-networking: %w", err)
+		}
+		if _, err := c.CoreV1().Nodes().Patch(context.TODO(), node.Name, types.StrategicMergePatchType, []byte(fmt.Sprintf(`{"metadata":{"annotations":%s}}`, raw)), metav1.PatchOptions{}); err != nil {
```

Comment:
Is it possible to do a patch just once? With this we will end up patching the node object twice here.

Reply:
maybe @aojea has some ideas, but Annotation have to use StrategicMergePatchType so we could atomically update only MN annotations, and then Capacity has to use JSONPatchType so that we could add and **remove** items.

Reply:
can you use an Update instead of a Patch?

PUT vs PATCH thing

Reply:
hmm, but why 
```
			"op":    "replace",
			"path":  "/metadata/annotations",
			"value": node.Annotations,
```

does not work for you?

Reply:
it replaces whole Annotation, and when any change is updated by other controller within the time this is processed, we overwrite that changes (happen to me w/o this change).